addons/dragon/data/M47mis.rvmat

```diff
@@ -0,0 +1,81 @@
+class StageTI {
+    texture="z\ace\addons\dragon\data\M47_TI_CA.tga";
+};
+ambient[]={0.98899996,1,0.958,1};
+diffuse[]={0.98899996,1,0.958,1};
```

Comment:
Manually typing that could make it alot nicer. You can see that these are computer generated on the terrible numbers.